### PR TITLE
Update client docs - 3rd option docker compose

### PR DESCRIPTION
**Pull request by Audiosutras**
This PR is to add a 3rd option, `docker-compose` to client deployment.

### CODE REVIEW

**Comment by Audiosutras** [comment]
I did not mention port forwarding from the router to make MWA reachable from the open internet. I would uncomfortable recommending this to an end user with minimum knowledge.

**Comment by binaryoverload** [comment]
After some research, it turns out that you can use a build context from a Git URL i.e.

```yml
build:
  context: https://github.com/movie-web/movie-web.git
```

So I think the best thing to do is include a stripped-down version of the docker-compose.yml file we have in the repo in the docs, with only the common args

That way people don't have to clone the repo

**Comment by Audiosutras** [comment]
@binaryoverload 😛 `dbDVwrTqQC3JxFmS4Uq5`

**Comment by Audiosutras** [comment]
> After some research, it turns out that you can use a build context from a Git URL i.e.
> 
> ```yaml
> build:
>   context: https://github.com/movie-web/movie-web.git
> ```
> 
> So I think the best thing to do is include a stripped-down version of the docker-compose.yml file we have in the repo in the docs, with only the common args
> 
> That way people don't have to clone the repo

Great to know! Changed in docs.

File: content/3.client/1.deploy.md [hunk 1]
```diff
@@ -43,3 +43,83 @@ title: 'Deploy'
...
+
+1. Make sure `movie-web` is your current working directory and run:
+
+```bash
+# Stop the running container
+$ docker compose stop
+# Remove the stopped container
+$ docker compose rm
+# Re-build the image and start the container
+$ docker compose up -d
```
Comment (binaryoverload) [inline content/3.client/1.deploy.md:124]: I don't think you need to stop and remove, I think adding `--build` to the `up` command will suffice

File: content/3.client/1.deploy.md [hunk 2]
```diff
@@ -43,3 +43,83 @@ title: 'Deploy'
...
+      #   CORS_PROXY_URL: ""
+      #   DMCA_EMAIL: ""
+      #   NORMAL_ROUTER: "false"
+      #   BACKEND_URL: ""
+      #   HAS_ONBOARDING: "false"
+      #   ONBOARDING_CHROME_EXTENSION_INSTALL_LINK: ""
+      #   ONBOARDING_PROXY_INSTALL_LINK: ""
+      #   DISALLOWED_IDS: ""
+      #   CDN_REPLACEMENTS: ""
+      #   TURNSTILE_KEY: ""
```
Comment (binaryoverload) [inline content/3.client/1.deploy.md:76]: Maybe just have a couple in here as examples, and then link to the config section for the rest? 

Might be worth updating the config page to also mention about these

**Review by binaryoverload — CHANGES_REQUESTED** [review]

File: content/3.client/1.deploy.md [hunk 3]
```diff
@@ -43,3 +43,83 @@ title: 'Deploy'
...
 9. Congrats! You have your own version of movie-web hosted.
+
+## Method 3 - Docker Compose - Home Network
+
+This method is meant for those using a desktop device or single board computer with a minimum of 4GB of RAM such as a [Raspberry Pi](https://www.raspberrypi.com/) to run movie-web on there home network for network connected devices.
+
+1. Ensure you have [docker](https://docs.docker.com/get-docker/) installed. In a newly created directory called `movie-web` create a file called `docker-compose.yaml`. Paste the contents of the code block below into this file.
+
+```yaml
+version: "3.8"
```
Comment (binaryoverload) [inline content/3.client/1.deploy.md:54]: It shouldn't cause a problem, but I generally specify a version like `3.2` to ensure maximum compatibility 

```suggestion
version: "3.2"
```

**Review by binaryoverload — CHANGES_REQUESTED** [review]

File: content/3.client/1.deploy.md (hunk 1 above)
Comment (Audiosutras) [reply content/3.client/1.deploy.md:124]: :heavy_check_mark:

File: content/3.client/1.deploy.md (hunk 2 above)
Comment (Audiosutras) [reply content/3.client/1.deploy.md:76]: See Step 5.

Comment (Audiosutras) [reply content/3.client/1.deploy.md:76]: Reduced to tmdb, proxy, backend_url

**Review by binaryoverload — APPROVED** [review]

**Pull request merged**